crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -677,7 +678,7 @@ impl PgManager {
             paid_address: None,
             input_object: None,
             changed_object: None,
-            transaction_ids: Some(digests.iter().map(|x| x.to_string()).collect::<Vec<_>>()),
+            transaction_ids: Some(digests.iter().map(|d| Digest::from(*d)).collect()),
```

Comment:
Maybe a dumb suggestion
```suggestion
            transaction_ids: Some(digests.into_iter().map(Digest::from).collect()),
```